internal/cli/atlas_events_list.go

```diff
@@ -26,15 +26,16 @@ import (
 )
 
 type atlasEventsListOpts struct {
-	globalOpts
```

Comment:
this command wasn't using any of `globalOpts` functions so no need to compose it here and I'll just use the properties